Doc/library/pyclbr.rst

```diff
@@ -39,78 +39,77 @@ modules.
    path.
 
 
-.. _pyclbr-class-objects:
+.. _pyclbr-object-objects:
 
-Class Objects
--------------
-
-The :class:`Class` objects used as values in the dictionary returned by
-:func:`readmodule` and :func:`readmodule_ex` provide the following data
-attributes:
-
-
-.. attribute:: Class.module
+Object Objects
+--------------
+The class :class:`Object` is the base class for the classes
+:class:`Class` and :class:`Function`. It provides the following
+data members:
```

Comment:
move;;+The class :class:`Object` is the base class for the classes
+:class:`Class` and :class:`Function`. It provides the following
+data members: